src/v1/components/inputs/Button/Button.tsx

```diff
@@ -33,7 +35,7 @@ const Button: React.FC<ButtonProps> = ({
   if (variant === "link") return <LinkButton {...buttonProps} />;
 
   return (
-    <MUIButton variant="text" color="primary" {...buttonProps}>
```

Comment:
👍 

We should warn people that using DS components with anything except defaults carries some risk,

`<Button />` is better than `<Button color="blah" />`

Reply:
Yeah, that's a good idea for when we have a more solid foundation of components and their use cases